
================================================================================
CustomizationFailed
================================================================================


.. describe:: Extended by
    
    :py:class:`~pyvisdk.do.customization_linux_identity_failed.CustomizationLinuxIdentityFailed`,
    :py:class:`~pyvisdk.do.customization_network_setup_failed.CustomizationNetworkSetupFailed`,
    :py:class:`~pyvisdk.do.customization_sysprep_failed.CustomizationSysprepFailed`,
    :py:class:`~pyvisdk.do.customization_unknown_failure.CustomizationUnknownFailure`
    
.. describe:: Since
    
    VI API 2.5
    
.. describe:: Extends
    
    :py:class:`~pyvisdk.mo.customization_event.CustomizationEvent`
    
.. class:: pyvisdk.do.customization_failed.CustomizationFailed
    